main_app/tests/test_app.py

```diff
@@ -21,41 +21,41 @@ def test_can_reformat_json_object(self):
-        """Can make an API call and expect specific attributes (run local server first)"""
-        data = requests.get('http://localhost:8000/api/coordinates="39.7392, -104.9903"').json()
-        self.assertEqual(type(data), list)
-
-        self.assertIn('formatted_address', data[0])
-        self.assertIn('geometry', data[0])
-        self.assertIn('name', data[0])
-        self.assertIn('opening_hours', data[0])
-        self.assertIn('photos', data[0])
-        self.assertIn('rating', data[0])
-
-    def test_can_make_map_API_call_and_save_a_park(self):
-        """Can make an API call and expect specific attributes (run local server first)"""
-        params = {
-        "name": "Fake Park",
-        "formatted_address": "1234 Fake Street",
-        "opening_hours": "never",
-        "photo": "too good for photos",
-        "rating": "100 mofo"
-        }
-        data = requests.post('http://localhost:8000/api/park/create/', params)
-        self.assertEqual(type(data), requests.models.Response)
-
-        result = requests.get('http://localhost:8000/api/park/all')
+    # def test_can_make_map_API_call_with_city_state(self):
+    #     """Can make an API call and expect specific attributes (run local server first)"""
+    #     data = requests.get('http://localhost:8000/api/coordinates="miami,fl"').json()
+    #     self.assertEqual(type(data), list)
+    #
+    #     self.assertIn('formatted_address', data[0])
+    #     self.assertIn('geometry', data[0])
+    #     self.assertIn('name', data[0])
+    #     self.assertIn('opening_hours', data[0])
+    #     self.assertIn('photos', data[0])
+    #     self.assertIn('rating', data[0])
+    #     # code.interact(local=dict(globals(), **locals()))
+    #
+    # def test_can_make_map_API_call_with_coordinates(self):
+    #     """Can make an API call and expect specific attributes (run local server first)"""
+    #     data = requests.get('http://localhost:8000/api/coordinates="39.7392, -104.9903"').json()
+    #     self.assertEqual(type(data), list)
+    #
+    #     self.assertIn('formatted_address', data[0])
+    #     self.assertIn('geometry', data[0])
+    #     self.assertIn('name', data[0])
+    #     self.assertIn('opening_hours', data[0])
+    #     self.assertIn('photos', data[0])
+    #     self.assertIn('rating', data[0])
+    #
+    # def test_can_make_map_API_call_and_save_a_park(self):
+    #     """Can make an API call and expect specific attributes (run local server first)"""
+    #     params = {
+    #     "name": "Fake Park",
+    #     "formatted_address": "1234 Fake Street",
+    #     "opening_hours": "never",
+    #     "photo": "too good for photos",
+    #     "rating": "100 mofo"
+    #     }
+    #     data = requests.post('http://localhost:8000/api/park/create/', params)
+    #     self.assertEqual(type(data), requests.models.Response)
+    #
+    #     result = requests.get('http://localhost:8000/api/park/all')
```

Comment:
are they commented out because the tests are not passing?

Reply:
Oh crap. They do pass if the server is running, I didn't want to keep running those tests cause they passed. My bad. I'll uncomment them now. Good catch